R/standardise_titles.R

```diff
@@ -35,7 +35,7 @@ standardise_titles <- standardize_titles <- function(s, strict = FALSE, api_key
   out <- vapply(strsplit(s, split = " "), cap, "", USE.NAMES = !is.null(names(s)))
 
   if (!is.null(api_key)) {
-    qData::depends("cld2", "translateR")
+    qCreate::depends("cld2", "translateR")
```

Comment:
Isn't `depends()` in qData?

Reply:
The function is in qCreate now, it was moved along with the package_tools script. Would you like me to move it back to qData? And, if so, would `retain()` be in qCreate or qData?